lib/ansible/modules/crypto/luks_device.py

```diff
@@ -246,8 +296,8 @@ def __init__(self, module):
         self._module = module
         self._lsblk_bin = self._module.get_bin_path('lsblk', True)
 
-    def _run_command(self, command):
-        return self._module.run_command(command)
+    def _run_command(self, command, **kwargs):
```

Comment:
It is better to only add the options you need, and not have a general `**kwargs` catch-all. Here you only need `data`, right? Better add it explicitly:
```suggestion
    def _run_command(self, command, data=None):
```